### PR TITLE
Implement MST protocol

### PR DESCRIPTION
**Pull request by tkindy**

### CODE REVIEW

File: 3700bridge [hunk 1]
```diff
@@ -84,24 +84,46 @@ class Bridge(object):
...
+            self.root_cost = new_cost
+            self.next_bridge = new_next
+            self.root_bpdu = low_bpdu
+
+    def update_root_port(self, root_port: int):
         self.root_port = root_port
-        self.root_cost = root_cost
         print('Root port: {}/{}'.format(serialize_address(self.bridge_id), root_port))
 
+        self.reset_routing_table()
```
Comment (aem) [inline 3700bridge:146]: you should only reset if any of those changed. if nothing changes we shouldn't reset the routing table

Comment (aem) [reply 3700bridge:146]: ah wait. misread this.

Comment (aem) [reply 3700bridge:146]: although i think we should be flushing the routing table here: https://github.com/aem/cs3700-project2/pull/3/files/8c2fdd6465d69fbe424ff3d5c95f93f98f6f5495#diff-e9f2bd50fd95eef919abd747ef089a7aR116

**Comment by aem** [comment]
is this PR or master the current source of truth?

**Comment by tkindy** [comment]
Neither, refer to #5

**Pull request closed without merging**